gensim/models/ldamodel.py

```diff
@@ -900,10 +902,27 @@ def get_document_topics(self, bow, minimum_probability=None):
         if is_corpus:
             return self._apply(corpus)
 
-        gamma, _ = self.inference([bow])
+        gamma, phis = self.inference([bow], collect_sstats=True)
         topic_dist = gamma[0] / sum(gamma[0])  # normalize distribution
-        return [(topicid, topicvalue) for topicid, topicvalue in enumerate(topic_dist)
-                if topicvalue >= minimum_probability]
+
+        document_topics = [(topicid, topicvalue) for topicid, topicvalue in enumerate(topic_dist)
+                    if topicvalue >= minimum_probability]
+        if not per_word_topics:
+            return document_topics
+        else:
+            word_phi = [] # contains word and corresponding topic
+            for word_type, weight in bow:
+                phi_values = [] # contains phi values for each topic
+                for topic_id in range(0, self.num_topics):
+                    if phis[topic_id][word_type] >= minimum_probability:
```

Comment:
Just a note: if the user does not enter a minimum_phi_probability, it ends up taking `self.minimum_probability`'s value.